analysis/webservice/algorithms/doms/ResultsStorage.py

```diff
@@ -285,7 +285,8 @@ def __enrichPrimaryDataWithMatches(self, id, dataMap, trim_data=False):
                     dataMap[row.primary_value_id]["matches"] = []
                 dataMap[row.primary_value_id]["matches"].append(entry)
             else:
-                print(row)
+                # print(row)
+                pass
```

Comment:
@skorper @RKuttruff Did you get a chance to figure out why we keep reaching this else statement?

Reply:
Yes, I think I shared it in our discussion on slack

Reply:
@ngachung 
> It’s in the pagination PR
[https://github.com/apache/incubator-sdap-nexus/blob/b1aa0910909ad8299a62bad6f235cc[…]0c65e6383/analysis/webservice/algorithms/doms/ResultsStorage.py](https://github.com/apache/incubator-sdap-nexus/blob/b1aa0910909ad8299a62bad6f235cc80c65e6383/analysis/webservice/algorithms/doms/ResultsStorage.py#L305-L313)
Gets some results for the execution ID, but then in lines 310-312 the data map is only for the subset of rows that match the page
[https://github.com/apache/incubator-sdap-nexus/blob/b1aa0910909ad8299a62bad6f235cc[…]0c65e6383/analysis/webservice/algorithms/doms/ResultsStorage.py](https://github.com/apache/incubator-sdap-nexus/blob/b1aa0910909ad8299a62bad6f235cc80c65e6383/analysis/webservice/algorithms/doms/ResultsStorage.py#L292-L298)
Here a very similar query is run, but when the returned rows are iterated over, it iterates over the whole Cassandra result, not just the rows that match the page

Reply:
NVM, I think this PR is no longer needed as I removed this in a separate PR: https://github.com/apache/incubator-sdap-nexus/blob/master/analysis/webservice/algorithms/doms/ResultsStorage.py#L293

Reply:
@ngachung To summarize, since we are only querying for a single page of data rather than ALL rows for a given execution ID, not all secondary rows with a matching execution ID will be used because they may be on a different page.

Reply:
Thanks @skorper, I'll close it